### PR TITLE
Adding the checksum project to transifex pipline.

### PR DESCRIPTION
**Pull request by westberliner**

### CODE REVIEW

**Comment by rakekniven** [comment]
Hello @westberliner ,

have you added user "nextcloud-bot" to your repo and gave it write permissions?

File: translations/config.json [hunk 1]
```diff
@@ -125,7 +125,8 @@
         "SergeyMosin appointments",
         "stjosh auto_groups",
         "sualko cloud_bbb",
-        "tacruc gpgmailer"
+        "tacruc gpgmailer",
+		"westberliner checksum"
```
Comment (rakekniven) [inline translations/config.json:129]: Please fix indention. Tabs to spaces.

**Review by rakekniven — CHANGES_REQUESTED** [review]
I will merge this as soon as you confirmed the bot invitation. See my comment at your PR.

**Comment by westberliner** [comment]
> Hello @westberliner ,
> 
> have you added user "nextcloud-bot" to your repo and gave it write permissions?

Hi,

I invited the bot. Is this the correct way?
Screenshot 2021-12-05 at 10 30 20

**Comment by rakekniven** [comment]
> > Hello @westberliner ,
> > have you added user "nextcloud-bot" to your repo and gave it write permissions?
> 
> Hi,
> 
> I invited the bot. Is this the correct way? Screenshot 2021-12-05 at 10 30 20

Which role did you assign to the bot?

File: translations/config.json (hunk 1 above)
Comment (westberliner) [reply translations/config.json]: Should be fixed. Somehow the browser changed spaces to tabs. Sry for that.

**Review by rakekniven — APPROVED** [review]

**Comment by westberliner** [comment]
Thx for the merge. I don't see any roles to assign I think because this repo is not under an organization.

**Comment by rakekniven** [comment]
Ok, as soon as the invitation will be accepted, it should show up as collaborator. Will be tomorrow I think.

**Pull request merged**